.gitignore

```diff
@@ -0,0 +1 @@
+.idea
```

Comment:
It's actually possible now to commit `.idea` to help with making standards adherence easier. You just have to exclude certain files. Here is what we have excluded on some of our other projects:
```
###> PhpStorm ###
.idea/**/workspace.xml
.idea/**/tasks.xml
.idea/dictionaries
.idea/httpRequests
# Sensitive or high-churn files:
.idea/**/dataSources/
.idea/**/dataSources.ids
.idea/**/dataSources.xml
.idea/**/dataSources.local.xml
.idea/**/sqlDataSources.xml
.idea/**/dynamic.xml
.idea/**/uiDesigner.xml
# Gradle:
.idea/**/gradle.xml
.idea/**/libraries
# File-based project format:
*.iws
###< PhpStorm ###
```

Reply:
@dd-zen @h2so3-id ^^
No need to fix. Just an FYI.